impeller/typographer/backends/skia/text_render_context_skia.cc

```diff
@@ -152,14 +152,14 @@ static std::optional<SkBitmap> CreateAtlasBitmap(const GlyphAtlas& atlas,
 
 static std::shared_ptr<Texture> UploadGlyphTextureAtlas(
     std::shared_ptr<Allocator> allocator,
-    const SkBitmap& bitmap,
+    std::shared_ptr<SkBitmap> bitmap,
     size_t atlas_size) {
   TRACE_EVENT0("impeller", __FUNCTION__);
   if (!allocator) {
     return nullptr;
   }
 
-  const auto& pixmap = bitmap.pixmap();
```

Comment:
There is a missing null check here now.

Reply:
Done. It looks like the `bitmap` here shouldn't be null, so I added a `FML_DCHECK` here.